config-model/src/main/java/com/yahoo/vespa/model/search/Tuning.java

```diff
@@ -303,6 +314,7 @@ public void getConfig(ProtonConfig.Summary.Log.Builder log) {
                         if (minFileSizeFactor!=null) log.minfilesizefactor(minFileSizeFactor);
                         if (chunk != null) {
                             chunk.getConfig(log.chunk);
+                            chunk.getConfig(log.compact);
```

Comment:
Nitpicking: I guess "chunk" is now more than just "chunk".

Reply:
Yes, but it is under the chunk tag in services so keeping that link here too.